xline-client/tests/common.rs

```diff
@@ -4,6 +4,10 @@ use xline_test_utils::Cluster;
 pub async fn get_cluster_client() -> Result<(Cluster, Client)> {
     let mut cluster = Cluster::new(3).await;
     cluster.start().await;
-    let client = Client::connect(cluster.addrs().clone(), ClientOptions::default()).await?;
+    let client = Client::connect(
+        cluster.addrs().iter().collect::<Vec<_>>(),
```

Comment:
Can we only pass the `iterator`` to avoid the heap allocation of `collect`?

Reply:
done, now connect accept a IntoIterator